docs/engineering/codex-repo-handoff-2026-04-24.md

```
@@ -48,6 +48,7 @@ If you are in a local checkout, prefer `AGENTS.md`, `.codex/START_HERE.md`, and
 37. `docs/engineering/zkai-attention-kv-stwo-native-d16-two-head-longseq-fused-softmax-table-gate-2026-05-10.md`
 38. `docs/engineering/zkai-attention-kv-stwo-native-two-head-seq32-fused-softmax-table-gate-2026-05-10.md`
 39. `docs/engineering/zkai-attention-kv-stwo-native-d16-two-head-seq32-fused-softmax-table-gate-2026-05-22.md`
+40. `docs/engineering/zkai-attention-kv-stwo-native-d32-four-head-longseq-fused-softmax-table-gate-2026-05-22.md`
 40. `docs/engineering/zkai-attention-kv-stwo-native-d32-two-head-fused-softmax-table-gate-2026-05-20.md`
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 41; Actual: 40; Style: 1/2/3

(MD029, ol-prefix)